resources/js/Pages/Announcement/List.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { usePage } from '@inertiajs/react';
+
+const List = () => {
```

Comment:
```suggestion
const List = ({ auth, announcements }) => {
```
Access props used in the List function's arguments. No need for `usePage().props`